docroot/themes/humsci/humsci_basic/src/scss/tools/_functions.animation.scss

```diff
@@ -11,11 +11,12 @@
   }
 
   @if ($is-valid-theme) {
-    $theme-animation-list: map-get($hb-animation-list, $hb-current-theme);
-    $hb-animation: $theme-animation-list[];
+    $animation-transition: map-get($hb-animation-list, $hb-current-theme);
```

Comment:
ack sorry. this should probably keep the naming you had before. I think I had a lot of ideas on the same comment yesterday, sorry!